### ISSUE TITLE
Pipeline constants

### ISSUE THREAD

**Issue by ChrOertlin** [Clinical-Genomics/cg#2811]
## Description

The current Pipeline constants model does not exclusively contain Pipelines, but rather it is a mix of workflows and pipelines. 
Also the enum "SARS_COV_2" points towards the "Mutant" pipeline, at least this is the folder naming on hasta for it. 

I think it makes sense the discuss to stratify and clarify the constants a bit better.

Low priority.

*Describe the problem/suggestion*

## Suggested solution

*Write here if there is a suggested way to solve the issue*

## This can be closed when

*Describe what needs to be done for this issue to be closed*

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by henrikstranneheim** [comment]
Rename Pipeline constant to Workflow:https://github.com/Clinical-Genomics/cg/pull/2885
Refactor pipeline to workflow for functions in cg: https://github.com/Clinical-Genomics/cg/pull/2894

**Comment by henrikstranneheim** [comment]
Pipeline is now workflow in cg, trailblazer and hermes. Resolved
